src/docs/concepts/meters.adoc

```diff
@@ -1,3 +1,3 @@
-Micrometer packs with a supported set of `Meter` primitives including: `Timer`, `Counter`, `Gauge`, `DistributionSummary`, `LongTaskTimer`, `FunctionCounter`, `FunctionTimer`, and `TimeGauge`. Different meter types result in a different number of time series metrics. For example, while there is a single metric that represents a `Gauge`, a `Timer` measures both the count of timed events and the total time of all events timed.
+Micrometer includes a supported set of `Meter` primitives, including `Timer`, `Counter`, `Gauge`, `DistributionSummary`, `LongTaskTimer`, `FunctionCounter`, `FunctionTimer`, and `TimeGauge`. Different meter types result in a different number of time series metrics. For example, while there is a single metric that represents a `Gauge`, a `Timer` measures both the count of timed events and the total time of all timed events.
 
-A meter is uniquely identified by its name and dimensions. We use the term dimensions and tags interchangeably, and the Micrometer interface is `Tag` simply because it is shorter. As a general rule it should be possible to use the name as a pivot. Dimensions allow a particular named metric to be sliced to drill down and reason about the data. This means that if just the name is selected, the user can drill down using other dimensions and reason about the value being shown.
+A meter is uniquely identified by its name and dimensions. We use the terms, "`dimensions`" and "`tags,`" interchangeably, and the Micrometer interface is `Tag` simply because it is shorter. As a general rule, it should be possible to use the name as a pivot. Dimensions let a particular named metric be sliced to drill down and reason about the data. This means that, if only the name is selected, you can drill down by using other dimensions and reason about the value being shown.
```

Comment:
Curly quotation marks again. Since it's not code, the comma goes inside the quotation marks. Sorry about the confusion. It's unfortunate that the way to get curly quotation marks looks like code inside quotation marks, but it's how Asciidoc works.